quaint/Cargo.toml

```diff
@@ -102,7 +127,7 @@ branch = "vendored-openssl"
 
 [dependencies.rusqlite]
 version = "0.29"
-features = ["chrono", "bundled", "column_decltype"]
+features = ["chrono", "column_decltype"]
```

Comment:
The `bundled` feature for `rusqlite` is enabled when the `sqlite-native` feature of `quaint` (implied by `native` and `all`) is turned on.